emarsys/api.py

```diff
@@ -111,8 +111,10 @@ def call(self, uri, method, params=None):  # noqa
         return result["data"]
 
     def _authentication_header_value(self):
-        now = datetime.datetime.now(self._tzinfo_obj)
-        created = now.replace(microsecond=0).isoformat()
+        created = (datetime.datetime.now(pytz.utc)
+                   .replace(microsecond=0)
+                   .isoformat())
```

Comment:
Ist jetzt hier der Unterschied, das das dem Konstruktor übergebene `tzinfo_obj` ignoriert und UTC hartkodiert wird? Wenn ja, dann sollte das so in der Commit-Message stehen.

Ggfs. wäre es angebracht dann auch `self._tzinfo_obj` zu entfernen.

Benutzen wir bei machtfit-oscar den gleichen falschen nicht-UTC-Code?

Reply:
Ja, der originale commit hat ` tzinfo_obj` ignoriert. Ich habe nun meinen Plan verworfen, den Ist-Stand möglichst zu übernehmen, denn hier können wir wahrscheinlich die Auswirkungen noch gut kontrollieren.

Deshalb nun ein ganz neuer commit, der den originalen commit referenziert, den Code leicht ändert, und `tzinfo_obj` entfernt. Da wir `tzinfo_obj` in machtfit/signup und machtfit/machtfit-oscar nie benutzt haben, ist diese Änderung aus Sicht der beiden repositories äquivalent zum anderen commit.

Damit schon die zweite Frage beantwortet: machtfit-oscar nutzt auch die Version ohne UTC, ist jedoch derzeit noch auf python-emarsys==0.6 gepinnt. Ich glaube, der UTC default ist auch für machtfit-oscar sinnvoll, aber das können wir später anpassen.